translations/base-en.yaml

```diff
@@ -1209,38 +1209,37 @@ tips:
     - You don't need to divide up items evenly for full efficiency.
     - Holding <b>SHIFT</b> will activate the belt planner, letting you place long lines of belts easily.
     - Cutters always cut vertically, regardless of their orientation.
-    - To get white mix all three colors.
+    - To get white, mix all three colors.
     - The storage buffer prioritises the left output.
     - Invest time to build repeatable designs - it's worth it!
-    - Holding <b>CTRL</b> allows to place multiple buildings.
+    - Holding <b>SHIFT</b> lets you place multiple buildings in one click.
```

Comment:
I think that refers to blueprints, but your version changes that to multiplace

Reply:
holding crtl during blueprint placement does nothing from experience

Reply:
Yeah, the old thing doesn't make much sense, though changing "in one click" to "at a time" may be better

Reply:
```suggestion
    - Holding <b>SHIFT</b> lets you drag to place buildings.
```